lib/phoenix_html/form.ex

```diff
@@ -402,7 +408,7 @@ defmodule Phoenix.HTML.Form do
       end
 
     if type == :text_input do
-      field = Atom.to_string(field)
+      field = to_string(field)
```

Comment:
Same here. We want to support only atom or strings, not any type that implements to_string.